### DISCUSSION TITLE
Is there any way one can see the Noobaa buckets as Directory and Object as files on the filesystem on which Noobaa is hosted.

### DISCUSSION THREAD

**Discussion by vamseekrishna25** [noobaa/noobaa-core#6453] [Q&A]
Is there any way one can see the Noobaa buckets as Directory and Object as files on the filesystem on which Noobaa is hosted.

**Comment by nimrod-becker** [comment]
Using normal buckets, no. The idea is that NooBaa does things like dedup (block level), compression and encryption. So the output of these is not a plain "file" but something you would need NooBaa in order to read.

However, if what you need is to use S3 but have a backing of a FS where you can also read those files. Namespace buckets sound like the way to go. Namespace buckets do not dedup, encrypt compress but rather save the objects in a "plain" mode on the backing storage. We are currently implementing more features/closing gaps to the NS FS capability, but you can start playing around with it. 

See this wiki page https://github.com/noobaa/noobaa-core/wiki/Namespace-FS:-Namespace-Resource-to-connect-to-shared-Filesystems

As always, would love to hear your thoughts on the wiki itserf and the behaviour of this feature in general

**Comment by vamseekrishna25** [comment]
hi I am trying to create_namespace_resource as suggested in the above link 
but I am getting following error:
noobaa api pool_api create_namespace_resource '{
>   "name": "fs1",
>   "nsfs_config": {
>       "fs_root_path": "/nsfs/fs1",
>       "fs_backend": "GPFS"
>   }
> }'
INFO[0001] ✅ Exists: NooBaa "noobaa"
INFO[0001] ✅ Exists: Service "noobaa-mgmt"
INFO[0001] ✅ Exists: Secret "noobaa-operator"
INFO[0001] ✅ Exists: Secret "noobaa-admin"
INFO[0001] ✈️  RPC: pool.create_namespace_resource() Request: map[name:fs1 nsfs_config:map[fs_backend:GPFS fs_root_path:/nsfs/fs1]]
WARN[0001] RPC: GetConnection creating connection to wss://localhost:34477/rpc/ 0xc00003b3b0
INFO[0001] RPC: Connecting websocket (0xc00003b3b0) &{RPC:0xc0000a1130 Address:wss://localhost:34477/rpc/ State:init WS:<nil> PendingRequests:map[] NextRequestID:0 Lock:{state:1 sema:0} ReconnectDelay:0s}
INFO[0001] RPC: Connected websocket (0xc00003b3b0) &{RPC:0xc0000a1130 Address:wss://localhost:34477/rpc/ State:init WS:<nil> PendingRequests:map[] NextRequestID:0 Lock:{state:1 sema:0} ReconnectDelay:0s}
ERRO[0001] ⚠️  RPC: pool.create_namespace_resource() Response Error: Code=INVALID_SCHEMA_PARAMS Message=INVALID_SCHEMA_PARAMS SERVER pool_api#/methods/create_namespace_resource
FATA[0001] ❌ INVALID_SCHEMA_PARAMS SERVER pool_api#/methods/create_namespace_resource
please let me know what went wrong
